File: LootMarket Backend/src/main/java/com/example/lootmarketbackend/dto/DettagliAstaDTO.java
package com.example.lootmarketbackend.dto;

public class DettagliAstaDTO {
    public int idAsta;
    public String emailCreatore;
    public String titolo;
    public String categoria;
    public double prezzoPartenza;
    public int anno;
    public int mese;
    public int giorno;
    public String descrizione;
    public double ultimaOfferta;
    public double sogliaMinima;
    public String tipoAsta;
    public Boolean offertaFatta;
    public String immagineAsta;
    public String nomeAutore;
    public String statusLegame;

    public DettagliAstaDTO(int idAsta,
                           String emailCreatore,
                           String titolo,
                           String categoria,
                           double prezzoPartenza,
                           int anno,
                           int mese,
                           int giorno,
                           String descrizione,
                           double ultimaOfferta,
                           double sogliaMinima,
                           String tipoAsta,
                           Boolean offertaFatta,
                           String immagineAsta,
                           String nomeAutore,
                           String statusLegame) {
        this.idAsta = idAsta;
        this.emailCreatore = emailCreatore;
        this.titolo = titolo;
        this.categoria = categoria;
        this.prezzoPartenza = prezzoPartenza;
        this.anno = anno;
        this.mese = mese;
        this.giorno = giorno;
        this.descrizione = descrizione;
        this.ultimaOfferta = ultimaOfferta;
        this.sogliaMinima = sogliaMinima;
        this.tipoAsta = tipoAsta;
        this.offertaFatta = offertaFatta;
        this.immagineAsta = immagineAsta;
        this.statusLegame = statusLegame;
        this.nomeAutore = nomeAutore;
    }
}

File: LootMarket/app/src/main/java/com/danilo/lootmarket/Network/dto/DettagliAstaDTO.kt
package com.danilo.lootmarket.Network.dto

data class DettagliAstaDTO(val idAsta: Int,
                           val emailCreatore: String,
                           val titolo: String,
                           val categoria: String,
                           val prezzoPartenza: Double,
                           val anno: Int,
                           val mese: Int,
                           val giorno: Int,
                           val descrizione: String,
                           val ultimaOfferta: Double,
                           val sogliaMinima: Int,
                           val tipoAsta: String,
                           val ultimaOffertaTua: Boolean,
                           val immagineAsta: String,
                           val nomeAutore: String,
                           val statusLegame: String)
